cscs-checks/system/io/ior_check.py

```diff
@@ -164,68 +169,21 @@ def __init__(self, fs_mount_point, ior_type):
         self.tags |= {'write'}
 
 
-# FIXME: This test is obsolete; it is kept only for reference.
-class IoMonchAcceptanceBase(IorCheck):
-    def __init__(self, fs_mount_point, ior_type, num_tasks):
-        super().__init__(fs_mount_point)
-        self.test_file = os.path.join(self.fs_mount_point, os.getenv('USER'),
-                                      'ior_write.dat')
-        self.valid_systems = ['monch:compute']
-        self.valid_prog_environs = ['PrgEnv-gnu']
-        self.num_tasks = num_tasks
-        self.num_tasks_per_node = 20
-        reference_by_num_tasks = {
-            40: {
-                'read_bw': (4343.66, -0.2, None),
-                'write_bw': (3625.23, -0.2, None)
-            },
-            80: {
-                'read_bw': (9208.83, -0.2, None),
-                'write_bw': (6725.39, -0.2, None)
-            },
-            160: {
-                'read_bw': (12365.53, -0.2, None),
-                'write_bw': (8073.84, -0.2, None)
-            },
-        }
-        self.reference = {
-            'monch:compute': reference_by_num_tasks[self.num_tasks]
-        }
-        self.tags = {'monch_acceptance'}
-
-
-# FIXME: This test is obsolete; it is kept only for reference.
-@rfm.parameterized_test(*(['/mnt/lnec', 'MPIIO', num_tasks]
-                          for num_tasks in [40, 80, 160]))
-class IorReadScratchMonchAcceptanceCheck(IoMonchAcceptanceBase):
-    def __init__(self, fs_mount_point, ior_type, num_tasks):
-        super().__init__(fs_mount_point, ior_type, num_tasks)
-        if ior_type == 'MPIIO':
-            self.executable_opts = ['-r', '-a MPIIO', '-B', '-E', '-F',
-                                    '-t 16m', '-b 8g', '-D 10', '-k',
-                                    '-o', self.test_file]
+@rfm.parameterized_test(['/scratch/snx1600tds'],
+                        ['/scratch/snx1600'],
+                        ['/scratch/snx3000tds'],
+                        ['/scratch/snx3000'],
+                        ['/users'],
+                        ['/scratch/shared/fulen']
+                        )
+class IorReadCheck(IorCheck):
+    def __init__(self, base_dir):
+        super().__init__(base_dir)
+        self.executable_opts.extend(['-r'])
```

Comment:
Rewrite this as follows (it's more readable):

```suggestion
        self.executable_opts += ['-r']
```